README.md

```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
-      refId: 'any',
-      scope: 'email'
-    }
-    
-    const requestParams = await silkeySdk.generateSSORequestParams(callerPrivateKey, callerParams)
+1.  Create a ethereum wallet for the program, save the private key
+2.  Save the application in Apollo with the generated wallet, visit https://apollo.silkey.io to do this
+3.  Store the private key in the .env variables for the application
 
-    // redirect user to provided silkey URL with all params from `requestParams` as query string
-```
+#### On Signin Page
 
-#### on callback page 
+```javascript
+import silkeySdk from "@silkey/sdk";
 
-##### default simple version
+// The needed data varaibles are:
+// redirectUrl: Where the user is redirected after auth
+// cancelUrl: Where the user is redirected if they cancel auth
+// scope: "email" or "id" email if you want access to the users email, otherwise id
+// refId: (optional) This data will be returned to the program after authentication, and can be used to track previous actions before signup
+// ssoTimestamp: (Optional) Time when params were generated, will be automatically generated if not present
 
-```
-    // token - get `token` from request parameters
+// Example:
+const requestParams = silkeySdk.generateSSORequestParams(privateKey, {
+  redirectUrl: "https://domain/callback",
+  cancelUrl: "https://domain/cancel",
+  scope: "email",
+  refId: "54321",
+});
 
-    const jwtPayload = silkeySdk.tokenPayloadVerifier(token)
+// Add the generated params to silkey url as queryString.
+// ATHENA_URL: The url of the Athena webpage that is authenticating the user
 
-    if (jwtPayload === null) {
-      // authorization failed
-    } else {
-      const {address, email, refId} = jwtPayload
-      // address - use this as ID of the user
+// Example:
+const silkeyRedirect = new URL((ATHENA_URL = "https://athena-staging.silkey.io"));
+Object.entries(requestParams).forEach(([key, param]) => {
+  silkeyRedirect.searchParams.append(key, param);
+});
 
-      // you are ready to go...
-    }
+// Rediect to silkeyRedirect
 ```
 
-##### version with additional silkey key check (recommended)
+#### On Callback Page
 
-- providerUri: web3 provider uri, we recommend using infura.io as a provider
-- registryAddress: Silkey smart contract registry address, see [#registryAddress](#registryAddress) for available addresses
+```javascript
+import silkeySdk from "@silkey/sdk";
 
-```
-    const silkeyPublicKey = await fetchSilkeyPublicKey(providerUri, registryAddress)
+// providerUri - A web3 provider URI. ie: 'https://infura.io/v3/:infuraId' register to infura.io to get infuraId
+// registryAddress: Address of silkey smart contract registry, see list of addresses in the registryAddress section of README.md
+const silkeyPublicKey = await silkeySdk.fetchSilkeyPublicKey(providerUri, registryAddress);
 
-    const jwtPayload = silkeySdk.tokenPayloadVerifier(token, silkeyPublicKey)
+const token = queryStringGetter("token");
```

Comment:
token can be send via POST or GET, lets developers care of getting it. btw. I think there is no form POST autosubmit on athena yet?